modules/distribution/resources/operation_policies/definitions/opaPolicy.gotmpl

```diff
@@ -0,0 +1,13 @@
+definition:
+  action: OPA
+  parameters:
+    requestGenerator: ""
```

Comment:
why don't we template this?

Reply:
We do not get this from UI, so this is not available in the policy spec.
If someone wants to have a custom requestGenerator generator, they have to create a new policy, by adding the class name here.